src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+$width-day: 100px;
+$height-day: 100px;
+$gap-width: 1px;
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  height: 100vh;
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-width;
+  padding: 10px;
+  width: calc($width-day * 7 + 10px * 2 + $gap-width * 6);
+}
+
+.calendar__day {
+  box-sizing: border-box;
```

Comment:
```suggestion
```

Reply:
Deleting this line creates visual and test issues
![image](https://github.com/mate-academy/layout_calendar/assets/21338158/3d7723e4-8524-420c-800b-f1bd3ddf04a3)